### PR TITLE
refactor: Remove guardrails for mainnet x-chain link claiming and cashouts

### PR DESCRIPTION
**Pull request by panosfilianos**
## Summary by CodeRabbit

- **New Features**
	- Enhanced error handling for KYC processes, providing specific messages and support options.
	- Improved confirmation flow for cashout transactions, adapting to user KYC status.
	- Refined logic for cross-chain transactions, allowing for better processing of token addresses.

- **Bug Fixes**
	- Corrected method call for token address processing to ensure proper functionality.

- **User Interface Adjustments**
	- Updated UI elements to reflect new error handling and confirmation steps.
	- Simplified conditions for displaying warning messages related to cross-chain support.

### CODE REVIEW

File: src/components/Cashout/Components/Confirm.view.tsx [hunk 1]
```diff
@@ -243,7 +243,7 @@ export const ConfirmCashoutView = ({
         }
 
         const route = await utils.fetchRouteRaw(
-            claimLinkData.tokenAddress.toLowerCase(),
+            claimLinkData.tokenAddress,
```
Comment (Hugo0) [inline src/components/Cashout/Components/Confirm.view.tsx:246]: Q: why is this removed? aren't we running the risk of wrong string comparison?

File: src/components/Cashout/Components/Initial.view.tsx [hunk 2]
```diff
@@ -40,11 +40,10 @@ export const InitialCashoutView = ({
...
 
-    const xchainAllowed = useMemo(
-        () =>
-            crossChainDetails.find((chain: any) => chain.chainId.toString() === selectedChainID.toString()) ||
-            selectedChainID === '1',
-        [crossChainDetails, selectedChainID]
+    const xchainAllowed = useMemo( (): boolean => {
+        return crossChainDetails.find((chain: any) => chain.chainId.toString() === selectedChainID.toString()) != undefined
+        },
+        [selectedChainID]
```
Comment (Hugo0) [inline src/components/Cashout/Components/Initial.view.tsx]: suggestion: would be good to have  a comment explaining what this code actually does

Comment (Hugo0) [reply src/components/Cashout/Components/Initial.view.tsx]: (ie: checking if we support x-chain on that chain)

File: src/components/Claim/Link/Offramp/Confirm.view.tsx [hunk 3]
```diff
@@ -62,7 +62,7 @@ export const ConfirmClaimLinkIbanView = ({
                 let route
                 try {
                     route = await utils.fetchRouteRaw(
-                        claimLinkData.tokenAddress.toLowerCase,
```
Comment (Hugo0) [inline src/components/Claim/Link/Offramp/Confirm.view.tsx:65]: LOL

File: src/components/Claim/Link/Offramp/Confirm.view.tsx [hunk 4]
```diff
@@ -62,7 +62,7 @@ export const ConfirmClaimLinkIbanView = ({
...
+                        claimLinkData.tokenAddress.toLowerCase(),
```
Comment (Hugo0) [inline src/components/Claim/Link/Offramp/Confirm.view.tsx]: TODO: Why is this lowercase kept but the other one removed?

File: src/components/Cashout/Components/Confirm.view.tsx (hunk 1 above)
Comment (panosfilianos) [reply src/components/Cashout/Components/Confirm.view.tsx:246]: This is a function spec moment at this point. The idea is that lowering takes place as part of fetchRouteRaw() logic. This way the function can support case agnosting addresses and .toLowerCase() is called in the function w/o need of pre-processing before.

File: src/components/Cashout/Components/Initial.view.tsx (hunk 2 above)
Comment (panosfilianos) [reply src/components/Cashout/Components/Initial.view.tsx]: On it!

File: src/components/Claim/Link/Offramp/Confirm.view.tsx (hunk 3 above)
Comment (panosfilianos) [reply src/components/Claim/Link/Offramp/Confirm.view.tsx:65]: yep :)

File: src/components/Claim/Link/Offramp/Confirm.view.tsx (hunk 4 above)
Comment (panosfilianos) [reply src/components/Claim/Link/Offramp/Confirm.view.tsx]: Please refer to the comment above: utils.fetchRouteRaw() is left to do the processing of the string inside its logic. No expectation of a case on the string params on case. This minimises errors (eg. someone forgets to minimize when passing - we can't type check that easily) and gives the fetchRouteRaw() more flexibility in the future, should another case is needed.

Comment (panosfilianos) [reply src/components/Claim/Link/Offramp/Confirm.view.tsx]: i.e. I'm pushing a commit that removes that as well!

File: src/components/Cashout/Components/Confirm.view.tsx (hunk 1 above)
Comment (Hugo0) [reply src/components/Cashout/Components/Confirm.view.tsx:246]: :saluting_face:

**Review by Hugo0 — APPROVED** [review]

**Pull request merged**